infrastructure/ansible/roles/nginx/defaults/main.yml

```diff
@@ -9,3 +9,19 @@ dhparams_pem_file: /etc/nginx/dhparam.pem
 nginx_allowed_ports:
   - 443
   - 80
+
+# 'lobby_proxy_pass' captures common selection logic for redirecting clients based on their
+# TripleA Version.
+#
+# Note: header names automatically have dashes converted to underscores.
+# Headers containing underscores are ignored entirely!
+lobby_proxy_pass: |
+    if ($http_triplea_version = "") {
```

Comment:
'if' is evil in NGINX apparently: https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/

I tried a solution that used `map` to createa `$lobbby_server` variable - though that did not work at all for the websocket `proxy_pass` configurations.